service/history/handler.go

```diff
@@ -693,7 +693,7 @@ func (h *Handler) DescribeHistoryHost(_ context.Context, req *historyservice.Des
 		}
 	}
 
-	itemsInCacheByIDCount, itemsInCacheByNameCount := h.namespaceRegistry.GetCacheSize()
+	itemsInCacheByIDCount, itemsInCacheByNameCount := h.namespaceRegistry.GetRegistrySize()
```

Comment:
Well, this needs to be renamed too. As well as `namespacespb.NamespaceCacheInfo` proto fields. I am ok to leave it as is though.